app/Css/Constants.hs

```diff
@@ -455,13 +455,14 @@ nodeFontSize :: Num a => a
 nodeFontSize = 12
 
 -- |Defines hybrid font size, 7 in pixels.
-hybridFontSize :: Num a => a
+hybridFontSize :: Integer a => a
```

Comment:
This isn't correct - you're mixing up a type class (`Num`) with a concrete type (`Integer`). Use `hybridFontSize :: Integer`. Also, this change wouldn't have compiled, so this is indicating that you aren't compiling your code before submitting a pull request, which you should make sure to always do.